functions/APIs/deals.js

```diff
@@ -1,9 +1,9 @@
 const { db } = require("../util/admin");
 
 /*
-display a list of all deals
+read a list of all deals
```

Comment:
Another really minor thing: Jessi has been asking Andrew to write comments as full sentences, with capital letters and punctuation.  Personally I think it's okay either way as long as it's consistent, but it's a little odd to have different styles between the frontend and backend code, so you might want to adopt that convention here too.